configurations/stress/c-s-throughput-cmds-i4i.yaml

```diff
@@ -0,0 +1,4 @@
+stress_cmd_w: "cassandra-stress write no-warmup cl=QUORUM duration=60m -schema 'replication(factor=3)' -mode cql3 native -rate threads=250 -pop seq=1..30000000"
```

Comment:
this file must start with `test_duration`

Reply:
it's not a test-case, it's a configuration file, i.e. it's only partial. (and we don't lint those files)

Reply:
The base conf file "test-cases/performance/perf-regression.100threads.30M-keys.yaml" has test duration.